streaming/base/partition/orig.py

```diff
@@ -52,6 +59,12 @@ def get_partitions_orig(num_samples: int,
 
     batch_size = batch_size or 1
 
+    # If drop_first isn't a multiple of num_physical_nodes, round down to make it divisible.
+    if drop_first % num_physical_nodes:
+        logger.warning('`drop_first` was not divisible by `num_physical_nodes`. Rounding it ' +
```

Comment:
Can you change `logger.warning` to `warnings.warn` and remove the changes that you made in pytest.ini file ?

Reply:
done